WhisperAPI/WhisperAPI/Services/Contexts.cs

```diff
@@ -22,11 +22,12 @@ public Contexts(DbContextOptions<Contexts> options, TimeSpan contextLifeSpan)
 
         // Get the Conversation associated to the chatkey,
         // create a new one if doesn't already exist
-        public ConversationContext this[Guid chatkey]
+        public virtual ConversationContext this[Guid chatkey]
         {
             get
             {
                 ConversationContext conversationContext = this.ConversationContexts
+                    .Include(x => x.MessagesSuggestions)
```

Comment:
Is this another restriction of Entity Framework ?

Reply:
Yes, it use lazy-loading by default.